plugins/pooling_function.cc

```diff
@@ -51,7 +52,10 @@ PoolingFunction(MldbServer * owner,
 {
     functionConfig = config.params.convert<PoolingFunctionConfig>();
 
-    set<string> validAggs = {"avg", "min", "max", "sum"};
+    SqlQueryFunctionConfig fnConfig;
+    fnConfig.query.stm.reset(new SelectStatement());
+
+    set<Utf8String> validAggs = {"avg", "min", "max", "sum"};
```

Comment:
These are the only aggregators we have which which make sense in the context... `count` and `pivot` don't make sense in this context and we don't have `stdev` :)